### DISCUSSION TITLE
What is the national dish of 🇺🇾?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#684] [Galaxy Trivia] [answered]
**Bonus**: List the main ingredients by using emojis.

**Comment by SANKARAMDAS** [comment]
Chivito is the national dish of Uruguay. It is a sandwich of sliced beefsteak (churrasco), mozzarella, ham, tomatoes, mayonnaise and black or green olives. A chivito commonly also includes bacon and fried or hard-boiled eggs.
🍖  🍅🍔

**Comment by peterbaccaro** [comment]
**Chivito** is the national dish of Uruguay. 

It typically includes the following ingredients:
🍞 Bread for the sandwich
🥩 Beef tenderloin or sirloin steak
🍖 Ham
🥓 Bacon
🧀 Mozzarella Cheese
🥬 Lettuce
🍅 Tomato
🫒  Olives
🥚 Hard Boiled Eggs
🫑 Grilled or sauteed red bell peppers
Mayonaise
Mustard

Some variations also include:
🧅 Onions
🥒 Pickles
🥚 Fried egg

**Comment by thuongtruong1009** [comment, marked as answer]
The national dish of Uruguay is "asado", which is a type of barbecue that typically consists of various cuts of beef cooked over wood or charcoal.

🐄🔥🥩🍴 = Asado

The main ingredients are:

Beef 🐄
Wood or charcoal 🔥
Various cuts of beef (such as ribs, steak, and sausage) 🥩
Utensils (such as forks and knives) 🍴
